Makefile

```diff
@@ -33,12 +33,16 @@ VERSION ?= master
 
 TAG_LATEST ?= false
 
+# The version of restic binary to be downloaded for power architecture
+RESTIC_VERSION ?= 0.9.4
```

Comment:
let's bump this to v0.9.5 to keep it consistent with our other images (can you confirm you have that version in your repo?)

Reply:
@skriss Yes we have that version in our repo

Reply:
Done